packages/react-composites/src/composites/common/CaptionsBannerMoreButton.tsx

```diff
@@ -0,0 +1,85 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IContextualMenuItem } from '@fluentui/react';
+import { ControlBarButtonProps, _StartCaptionsButton } from '@internal/react-components';
+import React from 'react';
+import { useMemo } from 'react';
+import { usePropsFor } from '../CallComposite/hooks/usePropsFor';
+import { buttonFlyoutIncreasedSizeStyles } from '../CallComposite/styles/Buttons.styles';
+import { useLocale } from '../localization';
+import { MoreButton } from './MoreButton';
+
+/** @private */
+export interface CaptionsBannerMoreButtonProps extends ControlBarButtonProps {
+  onCaptionsSettingsClick?: () => void;
+}
+
+/**
+ *
+ * @private
+ */
+export const CaptionsBannerMoreButton = (props: CaptionsBannerMoreButtonProps): JSX.Element => {
+  const localeStrings = useLocale();
+
+  const startCaptionsButtonProps = usePropsFor(_StartCaptionsButton);
```

Comment:
use the combination of useSelector + useHandler to avoid potential type leak in index file